R/read_price_data_internal_old.R

```diff
@@ -0,0 +1,41 @@
+#' Read in price data from csv and check that all expected columns are given.
+#'
+#' @description This function reads in price data using the old wide data format.
```

Comment:
is it possible to use a more concrete description than old wide data :) ? E.g. is it a particular year or source that will work with this?

Reply:
I wish I had a way to verify the exact year the data is taken from. But it just existed when I joined...